andela/bp-esa-frontend
# [Feature #160437219(Sub-Task) Implement filter with automation type

**What does this PR do?**
- Add the automation type filter to the report page
- Add automation types to the constants.js file.
- Add off boarding automation status to the mockReport.js

**Description of Task to be completed?**
User should be able to filter automation reports with automation type

**How should this be manually tested?**
- Clone and set up the project using the instruction in the README.
- Use $ git checkout ft-filter-report-with-type to check out the branch containing the changes of this pull request
- Use $ npm start to start the application
- Visit http://localhost:3000/ to see the report page.
- You should see the filter by automation type and also be able to interact with the different filter options

**What are the relevant pivotal tracker stories?**
[#160437219](https://www.pivotaltracker.com/story/show/160437219)

**Screenshots (if appropriate)**
<img width="1440" alt="screen shot 2018-09-26 at 09 25 04" src="https://user-images.githubusercontent.com/6957960/46070968-aeab2e80-c187-11e8-9428-5f131c6cef55.png">
<img width="1440" alt="screen shot 2018-09-26 at 12 29 25" src="https://user-images.githubusercontent.com/6957960/46071034-d4d0ce80-c187-11e8-8784-68132efa3827.png">
